job-service/src/main/java/com/hpe/caf/services/job/jobtype/DefaultDefinitionParser.java

```diff
@@ -115,18 +115,14 @@ public void setConfigurationProperties(
             final Map<String, String> configuration = new HashMap<>();
             for (final ConfigurationProperty property : properties) {
                 final String propertyName = property.getName(id, this);
-                final String propertyValue = appConfig.getJobTypeProperty(id, propertyName);
+                final String propertyValue = appConfig.getJobProperty(propertyName);
                 if (propertyValue == null) {
                     throw new InvalidJobTypeDefinitionException(
                         id + ": configuration is not available: " + propertyName);
                 }
                 // preserve property name case in the output, even though the configuration lookup
                 // ignores it
-                if (propertyName.equals("TARGET_PIPE") && propertyValue.equals("")) {
```

Comment:
I've removed this as I don't think we need this functionality anymore. (allowing empty string for target pipe). Looking at Josephs PR it seemed to be a way to signify that the target pipe is not set:

https://github.com/JobService/job-service/pull/97#issuecomment-505791033

We don't need this anymore as there is no need now to provide an env var for the target pipe if you don't want it set to anything.

Also, we don't know what the target_pipe config might be called now, as it could be named anything the user wants, so this test `propertyName.equals("TARGET_PIPE")` won't really work now.

Reply:
Great.